src/SingleBookingPage/bookingItem/Header.js

```diff
@@ -42,8 +43,25 @@ const Header = (props: Props) => {
         <div data-cy="booking-type">
           {booking.databaseId && (
             <Text type="secondary">
-              {isFuture ? 'Upcoming' : 'Past'} trip #&nbsp;
-              {formatBookingId(booking.databaseId)}
+              {isFuture ? (
+                <Trans
+                  t={__(
+                    'smartfaq.single_booking_page.header.booking_id.upcoming',
+                  )}
+                  html
+                  values={{
+                    booking_id: formatBookingId(booking.databaseId || 0) || '',
```

Comment:
then let's use camelCase everywhere that are not translation keys

Reply:
I think there's no snake case left (I mean none that I created)